packages/react-composites/tests/browser/utils.ts

```diff
@@ -11,7 +11,7 @@ export const dataUiId = (v: string): string => `[${DATA_UI_ID}="${v}"]`;
 const DATA_UI_ID = 'data-ui-id';
 const CONNECTION_STRING = process.env.CONNECTION_STRING ?? '';
 const PAGE_VIEWPORT = {
```

Comment:
can you leave a comment here that this should match the value set in the fixture.ts (if it needs to), or use the same value from a variable from a const file